include/envoy/runtime/runtime.h

```diff
@@ -205,6 +205,15 @@ class Snapshot {
    */
   virtual double getDouble(const std::string& key, double default_value) const PURE;
 
+  /**
+   * Fetch a boolean runtime key.
+   * @param key supplies the key to fetch.
+   * @param default_value supplies the value to return if the key does not exist or it does not
+   *        contain a boolean.
+   * @return bool the runtime value or the default value.
+   */
+  virtual bool getBoolean(absl::string_view key, bool default_value) const PURE;
```

Comment:
Can you please split this part of the change out into a separate PR?

Reply:
Done. See https://github.com/envoyproxy/envoy/pull/8476.